client/src/components/PageLayout.tsx

```diff
@@ -27,7 +28,12 @@ export function PageLayout(props: Props) {
 export function PageLayoutSimple(props: Props) {
   return (
     <Layout style={{ background: 'transparent' }}>
-      <Header isProfilePage title={props.title} username={props.githubId} courseName={props.courseName} />
+      <Header
+        isProfilePage={props.isProfilePage}
```

Comment:
where this prop is being set? you added it as possible prop, but I don't see it being used

Reply:
@Pulya10c please remove it completely. it was a mistake

Reply:
fixed